beta/src/components/MDX/Sandpack/sandpack-tsserver/tsserver.worker.ts

```diff
@@ -0,0 +1,477 @@
+import type {VirtualTypeScriptEnvironment} from '@typescript/vfs';
+import type {CompilerOptions} from 'typescript';
+import {ChannelServer} from './ChannelBridge';
+import ts from 'typescript';
+import {
+  createDefaultMapFromCDN,
+  createSystem,
+  createVirtualTypeScriptEnvironment,
+} from '@typescript/vfs';
+import type {Diagnostic} from '@codemirror/lint';
+
+const BUCKET_URL = 'https://prod-packager-packages.codesandbox.io/v1/typings';
+const TYPES_REGISTRY = 'https://unpkg.com/types-registry@latest/index.json';
+const wrappedPostMessage = (msg: any) => postMessage(msg);
+
+interface SerializedAction {
+  name: string;
+  data: ts.CodeFixAction;
+}
+
+export interface SerializedDiagnostic extends Diagnostic {
+  serializedActions: SerializedAction[];
+}
+
+type FetchedTypes = Record<string, {module: {code: string}}>;
+
+/**
+ * Fetch dependencies types from CodeSandbox CDN
+ */
+const fetchTypesFromCodeSandboxBucket = async ({
+  name,
+  version,
+}: {
+  name: string;
+  version: string;
+}): Promise<FetchedTypes> => {
+  try {
+    const url = `${BUCKET_URL}/${name}/${version}.json`;
+    const {files} = await fetch(url).then((data) => data.json());
```

Comment:
@DeMoorJasper, can we consume these types from the new CDN? I guess the new one is faster (maybe because it just doesn't export types 😛 )

Reply:
The new cdn removes types, probably not the reason it's faster as the old one also did this, types and code is seperated on the old packager.

The TS playground uses jsdelivr, so that would be one file at a time (quite a bit slower)

Reply:
Will keep as-is 👍🏻